lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -136,6 +137,7 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult, TPartitionType>
         public void OnNext(IDriverStarted value)
         {
             RequestUpdateEvaluator();
+            RequestMapEvaluators(_totalMappers);
```

Comment:
Good point! I can use EvaluatorBatchId to distinguish I think. If I know which evaluator is for master, we will not need to use _masterEvaluatorId == null in current GetContextConfigurationForEvaluatorById() to determine if we should submit a context for master.